internal/proto/controller/api/resources/sessions/v1/session.proto

```diff
@@ -9,106 +9,106 @@ import "controller/api/resources/scopes/v1/scope.proto";
 
   // Output only. Scope information for this resource.
   resources.scopes.v1.ScopeInfo scope = 30;
 
   // Output only. The time this resource was created.
-  google.protobuf.Timestamp created_time = 60 [json_name = "created_time"];
+  google.protobuf.Timestamp created_time = 60 [json_name = "created_time"]; // @gotags: `class:"public"`
 
   // Output only. The time this resource was last updated.
-  google.protobuf.Timestamp updated_time = 70 [json_name = "updated_time"];
+  google.protobuf.Timestamp updated_time = 70 [json_name = "updated_time"]; // @gotags: `class:"public"`
 
   // Version is used when canceling this Session to ensure that the operation is acting on a known session state.
-  uint32 version = 80;
+  uint32 version = 80; // @gotags: `class:"public"`
 
   // Output only. Type of the Session (e.g. tcp).
-  string type = 90;
+  string type = 90; // @gotags: `class:"public"`
 
   // Output only. After this time the connection will be expired, e.g. forcefully terminated.
-  google.protobuf.Timestamp expiration_time = 100 [json_name = "expiration_time"];
+  google.protobuf.Timestamp expiration_time = 100 [json_name = "expiration_time"]; // @gotags: `class:"public"`
 
   // Output only. The ID of the Auth Token used to authenticate.
-  string auth_token_id = 110 [json_name = "auth_token_id"];
+  string auth_token_id = 110 [json_name = "auth_token_id"]; // @gotags: `class:"public"`
 
   // Output only. The ID of the User that requested the Session.
-  string user_id = 120 [json_name = "user_id"];
+  string user_id = 120 [json_name = "user_id"]; // @gotags: `class:"public"`
 
   // Output only. The Host Set sourcing the Host for this Session.
-  string host_set_id = 130 [json_name = "host_set_id"];
+  string host_set_id = 130 [json_name = "host_set_id"]; // @gotags: `class:"public"`
 
   // Output only. The Host used by the Session.
-  string host_id = 140 [json_name = "host_id"];
+  string host_id = 140 [json_name = "host_id"]; // @gotags: `class:"public"`
 
   // Output only. The Scope of the Session.
-  string scope_id = 150 [json_name = "scope_id"];
+  string scope_id = 150 [json_name = "scope_id"]; // @gotags: `class:"public"`
 
   // Output only. The endpoint of the Session; that is, the address to which the worker is proxying data.
-  string endpoint = 160;
+  string endpoint = 160; // @gotags: `class:"public"`
 
   // Output only. The states of this Session in descending order from the current state to the first.
   repeated SessionState states = 170;
 
   // Output only. The current status of this Session.
-  string status = 180;
+  string status = 180; // @gotags: `class:"public"`
 
   // Output only. Worker information given to the client.
   repeated WorkerInfo worker_info = 190 [json_name = "worker_info"];
 
   // Output only. The certificate generated for the session. Raw DER bytes.
-  bytes certificate = 200;
+  bytes certificate = 200; // @gotags: `class:"public"`
```

Comment:
@jefferai is this secret?  I thought it provide auth/authz from the client to the worker..

Reply:
Nope!